lib/status/status.go

```diff
@@ -557,3 +572,91 @@ const (
 	// roleTag is the name of the tag containing node role
 	roleTag = "role"
 )
+
+// FromAlertManager collects alerts from the prometheus alertmanager deployed to the cluster
+func FromAlertManager(ctx context.Context, dnsAddr string) ([]*models.GettableAlert, error) {
+	client, err := httplib.GetPlanetClient(httplib.WithLocalResolver(dnsAddr))
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	transport := httptransport.NewWithClient(defaults.AlertmanagerServiceAddr, "/api/v2",
```

Comment:
What do you think about moving Alertmanager client creation to `lib/clients`? There are already a few there, this way we'll have a single place where API clients are created.

Reply:
I'm not sure what value wrapping the client would add?

Reply:
I don't know, I look at it from the perspective that all the layers of indirection and renaming things is where I struggle the most with this code base. So I want to connect to the alertmanager, I don't just create an alertmanager client, I have to go find the gravity client for the alert manager, that doesn't really do anything but wrap the client construction to give it a new name. So that's where I struggle to see the value add. Sure, someone needs to understand what they're doing, but even with a client it just means more code to explore to figure out how it works in order to be able to call it, and inevitably will require refactoring when someone wants to add behaviours.